filebeat/input/internal/procnet/procnet_test.go

```diff
@@ -30,7 +30,7 @@ func TestAddrs(t *testing.T) {
 			t.Errorf("unexpected error: %v", err)
 		}
 		if len(addr4) == 0 {
-			t.Errorf("expected addr in addr4 for IPv4 address: %v", addr6)
+			t.Errorf("expected addr in addr4 for IPv4 address: addr6 is %v", addr6)
```

Comment:
I'm trying to understand why printing `addr6` here. If the problem is having `addr4` empty, why not just state that?
```suggestion
			t.Error("expected addr in addr4 for IPv4 address, got none)
```

Reply:
@belimawr PTAL